modules/standard/IO.chpl

```diff
@@ -1439,13 +1436,13 @@ proc file.getPath(out error:syserr) : string {
   check();
   var ret:string;
   on this.home {
-    var tmp:c_string_copy;
-    var tmp2:c_string_copy;
+    var tmp:c_string;
+    var tmp2:c_string;
     error = qio_file_path(_file_internal, tmp);
     if !error {
       error = qio_shortest_path(_file_internal, tmp2, tmp);
     }
-    chpl_free_c_string_copy(tmp);
+    qio_free_string(tmp);
```

Comment:
This is just a macro in the runtime that says "This is temporary for now ..."

Can we use chpl_free_c_string instead? (and remove qio_free_string, which seems odd, especially since there's no corresponding qio_alloc_string)

Reply:
Probably... could that be part of another PR? It seems like there are a lot of little things we could do to clean up c_strings in general.

Reply:
Yeah, that sounds fine. I mentioned it only because this adds a second use of it (and the only other use of it is in this file too)